man/man8/zfs.8

```diff
@@ -3254,9 +3254,9 @@ The receiving system must also support this feature.
 Print verbose information about the stream package generated.
 This information includes a per-second report of how much data has been sent.
 .Pp
```

Comment:
nit:  This renders correctly, but I think it would be more correct to also pull the `.Pp` tag  tag out of the list.  It should be after the `.El` tag.